.github/workflows/pull_request.yml

```diff
@@ -72,6 +72,10 @@ jobs:
             ansible: "2.10"
             python: "3.6"
             toxpy: "36"
+          - test_type: packages-nodatabase
```

Comment:
naming is hard! The idea of this scenario is to test when pulp_database roles is not use

Reply:
"externaldb"?
It will not setup the database, but still run the migrations on the provided one, right?

Reply:
yes, it does run the migration.
externaldb is much better! thanks!